.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -245,10 +245,6 @@ jobs:
     - name: Create bundle output directory
       run: mkdir -p definitions/${{ github.event.pull_request.number }}
 
-    - name: Run Updater (single bundle)
```

Comment:
consider reverting... not needed anymore, but not quite related to this PR. Done because this is slow atm

Reply:
Created https://github.com/stackrox/stackrox/pull/12774 for this